template.yaml

```diff
@@ -50,44 +50,44 @@ Resources:
                   - logs:PutLogEvents
                 Resource: !Sub 'arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:*:*'
 
-  LmsGatewayAPI:
+  LmsCanvasGatewayAPI:
     Type: AWS::Serverless::Api
     Properties:
       StageName: v1
       EndpointConfiguration:
         Type: REGIONAL
       MethodSettings:
         - ResourcePath: /lms
-          HttpMethod: POST
+          HttpMethod: GET
```

Comment:
I think it might have to be a post here? I think api gateway always POST to the lambda function.

Reply:
have changed